src/main/java/org/jenkinsci/plugins/GithubSecurityRealm.java

```diff
@@ -244,6 +267,22 @@ public void marshal(Object source, HierarchicalStreamWriter writer,
             writer.setValue(realm.getOauthScopes());
             writer.endNode();
 
+            if (realm.getEmailDomains() != null) {
+                writer.startNode("emailDomains");
+                writer.setValue(realm.getEmailDomains());
+                writer.endNode();
+            }
+
+            writer.startNode("forceGithubEmail");
+            //TODO: Is there a better way to do this?
```

Comment:
Is there a better/preferred way to store a boolean value in the config file? If not and what I have is ok I'll remove the TODO comment.

Reply:
Check how other plugins save and load booleans.  I'll check as well.